vcli/vclirc

```diff
@@ -36,7 +36,7 @@ table_format = psql
 # Syntax Style. Possible values: manni, igor, xcode, vim, autumn, vs, rrt,
 # native, perldoc, borland, tango, emacs, friendly, monokai, paraiso-dark,
 # colorful, murphy, bw, pastie, paraiso-light, trac, default, fruity
-syntax_style = default
+syntax_style = monokai
```

Comment:
I like monokai as well, but the risk of setting it as a default is that it doesn't look great in a light background. 

Take a look at http://mycli.net/syntax and click on the image to toggle dark/light background.